library/Core/src/Juvix/Core/Base/Types/Base.hs

```diff
@@ -14,7 +14,23 @@ import Juvix.Library.Usage
 
 ------------------------------------------------------------------------------
 
-type Universe = Natural
+-- | a concrete universe level is just a natural number.
+type ConcUniverse = Natural
```

Comment:
Would it be too verbose to have `ConcUniverse` as a `newtype`?

Reply:
oops, yeah, thanks!

Reply:
it's only a synonym because the old `Universe` was. I'll make it a `newtype` now and whether it's a pain or not.